code/go/0chain.net/smartcontract/storagesc/config_settigns.go

```diff
@@ -448,8 +557,17 @@ func (conf *Config) set(key string, change string) error {
 		} else {
 			return fmt.Errorf("cannot convert key %s value %v to boolean: %v", key, change, err)
 		}
+	case smartcontract.Cost:
+		if key == SettingName[Cost] {
```

Comment:
I think the key value is `cost.*`, checking if its "cost" will always fail here.

Reply:
It will not, the type has been defined as Cost using enum.